### DISCUSSION TITLE
ngalert - Duplicate an entire alert definition

### DISCUSSION THREAD

**Discussion by alaricljs** [grafana/grafana#48836] [Alerting]
As noted in https://github.com/grafana/grafana/issues/39431 it would be great to be able to duplicate the entire definition of an Alert (8.0+ ngalerts) to then modify and save as a new Alert definition.  Perhaps a Save As... similar to the Dashboard method.

The noted issue was closed with a pointer on how to copy a query which is not a sufficient response.  I don't have a single alert that isn't a combination of at least 3 queries or expressions not to mention the default label set of 6 items.  My bigger alerts are ~10 queries+expressions.  

I really like the new alerts, there's just no good way to manage them yet.

**Comment by jantman** [comment]
This has now been implemented.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
